test/commands/autocomplete/create.test.ts

```diff
@@ -141,6 +141,7 @@ autocomplete:foo)
   _command_flags=(
     "--bar=-[bar for testing]:"
 "--baz=-[baz for testing]:"
+"--dangerous=-[\\\\\\\`with some potentially dangerous script\\\\\\\`]:"
```

Comment:
I know it needs 3 backlashes to escape twice, why 7? Other than that looks good.